### ISSUE TITLE
Can I make ui button in settings of app?

### ISSUE THREAD

**Issue by arpsyapathy** [RocketChat/Rocket.Chat.Apps-engine#504]
Hi!

Help me please.
I create app and make some settings in app via configuration.settings.
But I not found how make ui button in app settings. This is possible?  Configuration.settings don't have buttons.

Thank you advanced

**Comment by graywolf336** [comment]
You are correct, I do not believe there is a way to add a button to the settings page in the app. It does make sense to have one.

This is a feature request now @d-gubert
